packages/distros/rke2/zarf.yaml

```diff
@@ -0,0 +1,72 @@
+kind: ZarfInitConfig
+metadata:
+  name: "distro-rke2"
+  description: "Used to establish a new Zarf cluster"
+  architecture: amd64  # arm64 not yet supported by RKE2.  Remove when it is.
+
+components:
+  # AMD-64 version of the rke2 stack
+  - name: rke2
+    import:
+      path: common
+      name: rke2
+    only:
+      cluster:
+        architecture: amd64
+    files:
+      # Include the actual rke2 binary
+      - source: https://github.com/rancher/rke2/releases/download/v1.25.2%2Brke2r1/rke2.linux-amd64
+        shasum: 262c21a71b2e0381c1193800717a143bc80defac16541249546d6e208a22a6db
+        target: /usr/sbin/rke2
+        executable: true
+        # rke2 magic provides these tools when symlinking
+        symlinks:
+          - /usr/sbin/kubectl
+          - /usr/sbin/ctr
+          - /usr/sbin/crictl
+      # Transfer the rke2 images for containerd to pick them up
+      - source: https://github.com/rancher/rke2/releases/download/v1.25.2%2Brke2r1/rke2-images.linux-amd64.tar.zst
+        shasum: 5f09dd42e7876f8ca9485e2a06370bdf8baf586c29c373ef5ece7e62723fe46a
+        target: /var/lib/rancher/rke2/agent/images/rke2.tar.zst
+  # Placeholder for the ARM 64 Version when it's supported
+  - name: rke2
+    description: >
+      ****** READ THIS ARM 64 NOT SUPPORTED ********* 
+      rke2 is not currently supported on arm64 architecture, due to upstream dependencies. This functionality will be 
+      added when rke2 supports arm64 architecture.
+
+      If you would like to buid the amd64 version of this package on a arm64 based machine use the -a flag.
+
+      ex: `zarf package create . -a amd64`
+    import:
```

Comment:
do we need the import and files?  I think we just need the scripts part no?

Reply:
@jeff-mccoy would love your feedback here.  Left this commented out as a placeholder, but fine if it all goes away.

Reply:
You shouldn't have to re-import something already imported (or declared) by a component your importing.  However, I think for simplicities sake it might be worth forgoing the composable component here since there is no arm support anyway. The package-level `architecture` declaration at the top should be enough.